apps/www/components/ActionBar/index.js

```diff
@@ -315,7 +315,10 @@ const ActionBar = ({
       element: (
         <SubscribeMenu
           discussionId={isDiscussion && meta.ownDiscussion?.id}
-          subscriptions={document?.subscribedBy?.nodes}
+          subscriptions={document?.subscribedBy?.nodes?.filter(
+            (subscription) =>
+              !(subscription?.object?.id === document.id && !!meta.format),
```

Comment:
somehow I needed the conditionals `subscription?.object?.id` for it to work locally